pkg/k8s/apis/cilium.io/v2alpha1/lbipam_types.go

```diff
@@ -75,12 +75,6 @@ type CiliumLoadBalancerIPPoolSpec struct {
 	//
 	// +kubebuilder:validation:Optional
 	AllowFirstLastIPs AllowFirstLastIPType `json:"allowFirstLastIPs,omitempty"`
-	// Cidrs is a list of CIDRs comprising this IP Pool
-	// Deprecated: please use the `blocks` field instead. This field will be removed in a future release.
-	// https://github.com/cilium/cilium/issues/28590
-	//
-	// +kubebuilder:validation:Optional
-	Cidrs []CiliumLoadBalancerIPPoolIPBlock `json:"cidrs,omitempty"`
 	// Blocks is a list of CIDRs comprising this IP Pool
 	//
 	// +kubebuilder:validation:Optional
```

Comment:
unrelated to this PR, but we should think about having min/maxItems in CRD.

Reply:
Should be easy to add. I guess now that we have 1 list we can make it also required again (was optional before due to the aliased fields)

Reply:
starting with small values are easier IMO (e.g. 32), the reason I bring this up is for the case that we plan to ad for CEL validation later.

https://kubernetes.io/docs/reference/using-api/cel/
https://github.com/cilium/cilium/pull/32248#discussion_r1634906818

Reply:
Thanks, created an issue to track this https://github.com/cilium/cilium/issues/33203